appinventor/components/src/com/google/appinventor/components/runtime/ImageSprite.java

```diff
@@ -138,6 +138,9 @@ public void onDraw(android.graphics.Canvas canvas) {
             // to get the other right and bottom edges
             xinit + w / 2 + rotatedBitmap.getWidth() / 2,
             yinit + h / 2 + rotatedBitmap.getHeight() / 2);
+        // The value of alpha is hardcoded to 254 to avoid the black Background that occurs
+        // after a rotation ref: issue #400
```

Comment:
remove the comment line with the issue reference, since we can't assume that people working on App Inventor in the future will have access to that issue.